docs/application/web/tutorials/process/app-dev-process.md

```diff
@@ -256,22 +256,22 @@ After you have packaged your application, you are ready to certify and publish y
 
 To certify and publish your application:
 
--   Upload your mobile Web application to Tizen Store, your wearable Web application to the Samsung Galaxy Apps Store, or your TV Web application to the Samsung App Store for registration.
+-   Upload your mobile Web application to the official site for Tizen applications, your wearable Web application to the Samsung Galaxy Apps Store, or your TV Web application to the Samsung App Store for registration.
 
     After the application is uploaded, the application is signed as a certified application installer package and the `<Application_name>.wgt` archive format, which contains the distributor signature, is added by the applicable store.
 
 -   Submit your application to the applicable store for validation and publication.
 
     The store checks whether your application functions properly before publishing it.
 
-For detailed instructions on how to register your application and submit it for validation and publication in Tizen Store, see [Tizen Store Overview](https://developer.tizen.org/distribution/tizen-store-overview) and other subtopics within the **Distribution** section of the Tizen Developers site.
+For detailed instructions on how to register your application and submit it for validation and publication in the store, see [Store Overview](https://developer.tizen.org/distribution/tizen-store-overview) and other subtopics within the **Distribution** section of the Tizen Developers site.
```

Comment:
With the same reason of other comment, how about removing this line?
The link here is invalid now.